internal/controllers/utils/conditions.go

```diff
@@ -128,3 +129,15 @@ func IsClusterProvisionCompletedOrFailed(cr *provisioningv1alpha1.ProvisioningRe
 	}
 	return false
 }
+
+// IsSmoRegistrationCompleted checks if registration with SMO has been completed
+func IsSmoRegistrationCompleted(cr *inventoryv1alpha1.Inventory) bool {
+	condition := meta.FindStatusCondition(cr.Status.DeploymentsStatus.Conditions,
+		string(InventoryConditionTypes.SmoRegistrationCompleted))
+	if condition != nil {
```

Comment:
These  `if`s should be able to be on the same line, with a `&&`

Reply:
fixed